ext/date/php_date.stub.php

```diff
@@ -123,7 +123,10 @@ function timezone_name_from_abbr(string $abbr, int $utcOffset = -1, int $isDST =
 
 function timezone_offset_get(DateTimeZone $object, DateTimeInterface $datetime): int {}
 
-/** @refcount 1 */
+/**
+ * @return array<int|string, mixed>|false
```

Comment:
```suggestion
 * @return array<int, array>|false
```

Reply:
Sorry, I merged it too fast. I'll create a followup.